README.md

```diff
@@ -58,6 +58,11 @@ Currently our extension will activate when it detects an XML file. If it doesn't
 
 In a Theia editor, this file should be `.theia/settings.json` in your workspace, and for VS code it should be `.vscode/settings.json`. Once set, you can open any CNXML file and the extension should load.
 
+## Diagram of POET server/client/webview communication
+
+![poet-communication](./docs/poet-communication.png)
```

Comment:
I think this PNG file is missing from the PR currently.

Reply:
oops! added it now